frontend/src/views/WheresMyGene/components/Main/index.tsx

```diff
@@ -214,7 +227,13 @@ export default function WheresMyGene(): JSX.Element {
 
   const [forceOpen, setForceOpen] = useState(false);
 
-  const [isDownloading, setIsDownloading] = useState(false);
+  const [downloadStatus, setDownloadStatus] = useState<{
+    isLoading: boolean;
+    blur?: boolean;
```

Comment:
IIRC both image and CSV downloads require `blur`, so we probably don't need to specify `blur` anymore?